spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+feature 'Picture' do
+  # attach_file('Image', 'spec/files/images/coffee.jpg')
+
+  def register(user)
+    visit '/'
+    click_link 'Register'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    fill_in 'Password confirmation', with: user.password_confirmation
+    click_button 'Register'
+  end
+
+  let(:user) { build :user }
+
+  context 'when there are no pictures' do
+
+    scenario 'should have a prompt to add a picture' do
+      visit '/pictures'
+      expect(page).to have_link 'Add a picture'
+    end
+
+    scenario 'cannot add a picture if not logged in' do
+      visit '/pictures'
+      click_link 'Add a picture'
+      expect(page).to have_content 'You need to sign in or sign up before continuing.'
+      expect(page.current_path).to eq '/users/sign_in'
+    end
+
+    scenario 'can add a picture if logged in' do
+      register(user)
+      click_link 'Add a picture'
+      attach_file('picture[image]', File.join(Rails.root,'spec',"files", 'images', 'duck.jpg'))
+      fill_in 'Name', with: 'Kiss'
+      click_button 'Create Picture'
+      expect(page).to have_content 'Kiss'
+      expect(page).not_to have_content 'No pictures yet'
+    end
+
+  end
+
+  context 'when creating pictures' do
+
+    scenario "picture's name cannot be shorter than 2 characters" do
+      register(user)
+      visit '/pictures'
+      click_link 'Add a picture'
+      attach_file('picture[image]', File.join(Rails.root,'spec',"files", 'images', 'duck.jpg'))
```

Comment:
Metrics/LineLength: Line is too long. [95/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)